s3torchbenchmarking/utils/prepare_ec2_instance.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Script to prepare an EC2 instance for PyTorch benchmarks.
+
+set -euo pipefail
```

Comment:
To quote https://ivan.dev/articles/better-shell-scripts:

> - `-e` exits immediately if any simple command fails with a non-zero status. This avoids executing further commands that may have unintended consequences.
> - `-u` treats unset variables as an error when substituting, rather than substituting with an empty string. This helps catch typos in variable names.
> - `-o pipefail` returns a non-zero exit code if any command in the pipeline fails, not just the last one. This helps catch errors in a pipeline.